lib/rails_admin_import/formats/csv_importer.rb

```diff
@@ -0,0 +1,56 @@
+require 'csv'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.